TestFoundation/TestXMLDocument.swift

```diff
@@ -527,6 +528,22 @@ class TestXMLDocument : LoopbackServerTest {
         let expected = xmlString.lowercased() + "\n"
         XCTAssertEqual(expected, String(describing: document))
     }
+
+    func test_rootElementRetainsDocument() {
+        let str = """
+<?xml version="1.0" encoding="UTF-8"?>
+<plans></plans>
+"""
+
+        let data = str.data(using: .utf8)!
```

Comment:
It doesn't matter for this PR, but `let data = Data(str.utf8)` is nicer as there's no optional involved.

Reply:
Yes, definitely need a cleanup of the tests in `master` at some point.